browser-test/src/trusted_intermediary.test.ts

```diff
@@ -49,6 +49,41 @@ test.describe('Trusted intermediaries', {tag: ['@uses-fixtures']}, () => {
     await waitForPageJsLoad(page)
     await tiDashboard.expectDashboardContainClient(updatedClient)
   })
+  test('expect client info to be updated with empty emails', async ({
+    page,
+    tiDashboard,
+  }) => {
+    await loginAsTrustedIntermediary(page)
+    await tiDashboard.gotoTIDashboardPage(page)
+    await waitForPageJsLoad(page)
+    const client: ClientInformation = {
+      emailAddress: '',
+      firstName: 'Tony',
+      middleName: '',
+      lastName: 'Stark',
+      dobDate: '2021-06-10',
+    }
+    await tiDashboard.createClient(client)
+    await tiDashboard.expectDashboardContainClient(client)
+    await tiDashboard.updateClientTiNoteAndPhone(
+      client,
+      'Technology',
+      '4259746122',
+    )
+    await tiDashboard.expectSuccessAlertOnUpdate()
+    await validateScreenshot(
```

Comment:
suggestion (non-blocking):  I don't think we need a screenshot here since we're already checking that we got a success alert and that the client is in the client list.  In my opinion, it's best to avoid unnecessary screenshots because that just makes more to update every time we make a small UX change to this page.